docs/shape_batch_size_and_layout.md

```diff
@@ -31,17 +31,40 @@ In case the model can't be reshaped, it will remain in the original parameters a
 will get an error. The model server will also report such problem in the logs.
 
 # Changing model input/output layout
-**DEPRECATED** TODO: Suggest converting model to NHWC model (possible with Model Optimizer in 2022.1).
+Starting from 2022.1, Model Optimizer by default preserves the layout of original model. If the model accepted NHWC layout before conversion to IR format, it will preserved after conversion. Model Optimizer can also be used to add transposition step to change the layout to desired one. Models which process image data are usually exported with NCHW or NHWC layout. Image transformation libraries like OpenCV or Pillow use NHWC layout. To minimize amount of overhead caused by transposition operation, it is suggested to export your model with NHWC layout.
 
-OpenVINO models which process image data are generated via the model optimizer with NCHW layout. Image transformation libraries like OpenCV or Pillow use NHWC layout. This makes it required to transpose the data in the client application before it can be sent to OVMS. Custom node example implementations internally also use NHWC format to perform image transformations. Transposition operations increase the overall processing latency. Layout parameter reduces the latency by changing the model in runtime to accept NHWC layout instead of NCHW. That way the whole processing cycle is more effective by avoiding unnecessary data transpositions. That is especially beneficial for models with high resolution images, where data transposition could be more expensive in processing.<br><br>
+In case you already have model with NCHW layout and are not willing to re-export it, OpenVINO™ Model Server allows changing input/output layout at runtime with `--layout` parameter via CLI or `config.json`. Please note that it modifies the model by adding transposition operations as pre-processing step.
 
-Layout parameter is optional. By default layout is inherited from OpenVINO™ model. You can specify layout during conversion to IR format via Model Optimizer. You can also use this parameter for ONNX models.<br>
+Layout parameter is optional. By default layout is inherited from OpenVINO™ model. You can also use this parameter for ONNX models. In case no layout was specified during model export phase, the default layout OpenVINO™ Model Server sets is `N...`. This means that there is only one known dimension which is batch (`N`) and it is on first position.
```

Comment:
```suggestion
Layout parameter is optional. By default layout is inherited from OpenVINO™ model. You can use this parameter to adjust models both in ONNX and Intermediate Representation format. In case no layout was specified during model export phase, the default layout OpenVINO™ Model Server sets is `N...`. This means, there is known only the first dimension - the batch (`N`).
```